string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,17 @@
+/* exported isVowel */
+function isVowel(char) {
+  switch (char) {
+    case 'a':
+    case 'A':
+    case 'e':
+    case 'E':
+    case 'i':
+    case 'I':
+    case 'o':
+    case 'O':
+    case 'u':
+    case 'U':
+      return true;
+  }
+  return false;
+}
```

Comment:
If you convert `char` to lowercase, then you don't need to worry about checking both uppercase and lowercase.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```